bitnami/minio/values.yaml

```diff
@@ -2,6 +2,7 @@
 ## Global Docker image parameters
 ## Please, note that this will override the image parameters, including dependencies, configured to use the global value
 ## Current available global Docker image parameters: imageRegistry, imagePullSecrets and storageClass
+## Warning: All gateway features have been removed from Minio since upstream completely dropped this feature
```

Comment:
I think adding it to README.md is enough.

Reply:
➕1️⃣